### ISSUE TITLE
Same input resulting in different passwords

### ISSUE THREAD

**Issue by victormsf** [lesspass/lesspass#503]
Hi!
I've just started using LessPass today and, while trying it to decide if it would replace my current password manager (bitwarden), I've noticed that the same input results in different passwords. I've got **two** different passwords (password 1 and pw 2 below) in **three** different ways to use LessPass and this always with the same input (site, username and Master Password, the latter confirmed by the resulting 3 icons next to it being always the same):

1. In LessPass site - **password 1**;
2. In the browser extension, without logging-in LessPass - **password 2**;
3. In the browser extension, after signing-in to LessPass - **password 2**.

I've repeated this several times in different websites and with different browsers and kept getting the same results: the password in LessPass site was always different from the password I got in the browser extension. 

With this behaviour, LessPass fails to achieve part of what Guillaume wanted and that would be the answer to some questions he asked in "How does it work?":

> - How do I access a password on my parents’ computer without installing my password manager?
> - How do I access a password on my phone, without any installed app?

Because having at home a computer with LessPass extension in the browser and then at my parents' computer using LessPass website, I wouldn't be able to log-in where I wanted because I would be trying to log-in with a wrong password.

I really hope this issue will be addressed because I really like the concept of stateless password managers. I already have Master Password (https://masterpassword.app/) in my PC and smartphone, but I just use it in a couple of more **sensitive** websites because, unlike LessPass, it doesn't have browser extensions (even though it has a web app available through the website).

Best regards,
Victor.

**Comment by guillaumevincent** [comment]
Hello @victormsf,

so to summarize if you use the same site, username and master password on the web site and the web extension you got a different generated password right?

If true this is a big problem.

We have plenty of tests ensuring this is not the case.

What browser do you use?
What version of the web extension do you use?

Can you try on your side to test with site: '**a**', username: '**a**' and Master Password: '**a**' ?

Thanks

**Comment by victormsf** [comment]
Hello Guillaume,

When I reported it, it happened several times in Firefox, but not in Chromium. As of today, everything seems to be alright. Probably some of the input wasn't correct, but I had to be repeating the same error on the website, because the resulting incorrect password was always the same, and inputting everything correctly on the web extension.

Sorry for the inconvenience. Best regards.
